src/data/injections.js

```diff
@@ -438,6 +395,49 @@ const AVAILABLE_INJECTIONS = [
     },
     customFunc: "detectSwipeFix",
   },
+  {
+    id: "bug1625224",
+    platform: "all",
+    domain: "sixt-neuwagen.de",
+    bug: "1625224",
+    contentScripts: {
+      matches: ["*://*.sixt-neuwagen.de/*"],
+      js: [
+        {
+          file:
+            "injections/js/bug1625224-sixt-neuwagen.de-window-netscape-shim.js",
+        },
+      ],
+    },
+  },
+  {
+    id: "bug1630280",
+    platform: "android",
+    domain: "dominos.ch",
+    bug: "1630280",
+    contentScripts: {
+      matches: ["https://www.dominos.ch/*"],
+      css: [
+        {
+          file: "injections/css/bug1630280-dominos.ch-hide-overlay.css",
+        },
+      ],
+    },
+  },
+  {
+    id: "bug1629926",
+    platform: "desktop",
+    domain: "hangouts mic",
```

Comment:
Do we want to be a little more clear on the fix here, and say "GMail Hangouts microphone fix", or just say `mail.google.com` instead to be accurate to the domain?

Reply:
Yeah, good point, thanks!